content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -141,7 +141,8 @@ By default, all engine files are downloaded when you install Prisma CLI, copied
 `PRISMA_QUERY_ENGINE_BINARY` is used to set a custom location for your own query engine binary.
 
 ```terminal
-PRISMA_QUERY_ENGINE_BINARY=custom/my-query-engine-unix
+PRISMA_QUERY_ENGINE_BINARY=path/to/my/query-engine-<target>
```

Comment:
Thank you for catching this issue @samrith-s! 

For consistency reasons, can we preserve the use of `custom/`? 

We're already quite consistent in using this to indicate custom paths in several pages throughout the docs.

Reply:
Works, will do that!